driver-adapters/postgresql-node-basic/index.test.js

```diff
@@ -0,0 +1,105 @@
+const { handler } = require('./index')
+const { dependencies } = require('./package.json')
+
+jest.setTimeout(30000)
```

Comment:
Why is that needed, is it only needed in CI?

Reply:
Removed the timeout.

Reply:
Sometimes the test can be slower than 5 seconds default timeout
https://github.com/prisma/ecosystem-tests/actions/runs/6088467323/job/16519182818?pr=3901#step:8:409

I lowered it to 10 seconds.

Reply:
Probably because it's sequential and the latency is bad?